### PR TITLE
Add note about Cloud9 ownership 

### PR DESCRIPTION
**Pull request by ttrentler**
#### Which issue(s) this PR fixes: Add note about Cloud9 ownership #334 

Fixes #

#### Quality checks

- [x] My content adheres to the style guidelines
- [ ] I ran `make test` or `make e2e-test` and it was successful

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by ttrentler** [comment]
If approved, may  need to add the new page - [fix-cloud9.md](https://github.com/ttrentler/eks-workshop-v2/blob/main/website/docs/introduction/fix-cloud9.md) to the sidebar.  It is linked in to from the text on [ide.md](https://github.com/ttrentler/eks-workshop-v2/blob/main/website/docs/introduction/ide.md)

**Review by niallthomson — CHANGES_REQUESTED** [review]
Thanks for addressing this issue! Couple of content comments.

With regards to the sidebar my thoughts are that this page can live "off to the side" without being in the sidebar. I would suggest creating a directory `website/docs/misc` and moving the markdown file there. You can then update the link in `ide.md`.

File: website/docs/introduction/fix-cloud9.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+The above arn should be replaced with the arn of the user or role that needs access to the Cloud9 instance.
+
+    environment_id_from_arn
+
+The environment_id_from_arn should be replaced with the environment-id from the arn of the instance you want to manage. 
+The arn can be found by clicking on the instance name.  Everything after the last colon in the arn is the environment-id.
+
+![cloud9-arn](./assets/cloud9-arn.png)
+
+After entering the code wth the replaced text in the CLI you should now be able to access the Cloud9 Instance.
```
[the remark this answers is not in the document]
Comment (ttrentler) [reply website/docs/introduction/fix-cloud9.md]: typo fixed.

File: website/docs/introduction/ide.md [hunk 2]
```diff
@@ -13,10 +13,13 @@ When the main Cloud9 screen opens expand the menu on the left side of the screen
 
 ![Access Cloud9 service menu](./assets/menu.png)
 
-There will be a Cloud9 environment available, click the **Open** button to launch the IDE:
+There will be a Cloud9 environment named **eks-workshop** available, click the **Open** button to launch the IDE:
 
 ![Open the Cloud9 IDE](./assets/environment.png)
 
+*If you do not see the eks-workshop Cloud9 environment this is becuse it is owned by another IAM user. [Click here](./fix-cloud9.md) to see how to resolve the issue.*
```
[the remark this answers is not in the document]
Comment (ttrentler) [reply website/docs/introduction/ide.md]: Added the Tip - Thanks for the suggestion and the link!

**Comment by ttrentler** [comment]
Thanks for help getting these changes approved! 

With regards to the sidebar  - I moved the new Cloud9 content to a `misc ` sub-directory under `introduction` in the website folder and updated all the links.  If it makes more sense laying it out differently - that is fine.

**Comment by niallthomson** [comment]
Thanks for the fixes!

The suggestion for the sidebar was more to basically move that page to a place where it becomes "invisible" to the sidebar since I can't really think of a nice way to lay it out right now. My experience is that people tend to have an habit of navigating the workshop pages in sequence so having that page moved out of the way means they won't accidentally follow it unless they follow your note.

So instead of putting it in `website/docs/introduction/misc` just create a new directory at `website/docs/misc`. This means it won't show up in the sidebar.

**Comment by niallthomson** [comment]
@ttrentler the preview failure here may not be related to your changes, let me look in to it

**Comment by niallthomson** [comment]
@ttrentler should be resolved, please push a dummy change to your branch to force a new preview check

**Comment by ttrentler** [comment]
@niallthomson Still erroring on preview.

**Comment by niallthomson** [comment]
Would likely be fixed by a rebase, confirmed it works manually.

Thanks for the PR!

**Pull request merged**